docs/quick_start/README.md

```diff
@@ -1,29 +1,35 @@
 # Dragonfly Quick Start
 
-Dragonfly Quick Start document aims to help you to quick start Dragonfly journey. This experiement is quite easy and simplified.
+Dragonfly Quick Start document aims to help you to quick start Dragonfly journey. This experiment is quite easy and simplified.
 
 If you are using Dragonfly in your production environment to handle production image distribution, please refer to supernode and dfget's detailed production parameter configuration.
 
 ## Prerequisites
 
-All steps in this document are done on the same machine using the docker container, so make sure the docker container engine is installed and started on your machine. You can also refer to the documentation: [multi-machine deployment] (../userguide/multi_machines_deployment.md) to experience Dragonfly.
+All steps in this document are done on the same machine using the docker container, so make sure the docker container engine is installed and started on your machine. You can also refer to the documentation: [multi-machine deployment](../user_guide/multi_machines_deployment.md) to experience Dragonfly.
 
 ## Step 1: Deploy Dragonfly Server (SuperNode)
 
 ```bash
 docker run -d --name supernode --restart=always -p 8001:8001 -p 8002:8002 \
-    dragonflyoss/supernode:0.4.0 -Dsupernode.advertiseIp=127.0.0.1
+    dragonflyoss/supernode:0.4.1 -D --advertise-ip $(host-ip)
```

Comment:
* remove `-D`
* `$(host-ip)` is not clear for use, it cannot be executed in command line directly.
* If we use image 0.4.1, we can remove this option `--advertise-ip`, add `--node` to dfclient and remind user to use `--advertise-ip` if the `dfclient` cannot connect to `supernode`.

Reply:
Done. I have remove these two parameters.